faunadb/src/jepsen/faunadb/register.clj

```diff
@@ -82,4 +82,4 @@
      :checker (indy/checker
                 (checker/compose
                   {:timeline     (timeline/html)
-                   :linearizable (checker/linearizable)}))}))
+                   :linearizable (checker/linearizable {:model (model/cas-register)})}))}))
```

Comment:
You'll want to remove :model above, and use (cas-register 0) here. But yes, this is the right idea!